spec/models/user_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'default user details' do
+    let(:user) { User.new(name: 'Alex Doc', role: 2) }
```

Comment:
if the role is separated model, where creation of the role in this testcase?
like `let(:doctor_role) { Role.create!(anme: 'doctor') }` and than
`let(:user) { User.new(name: 'Alex Doc', role: doctor_role) }`